include/faabric/transport/MessageEndpointServer.h

```diff
@@ -22,27 +20,14 @@ class MessageEndpointServer
   public:
     MessageEndpointServer(int portIn);
 
-    /* Start and stop the server
-     *
-     * Generic methods to start and stop a message endpoint server. They take
-     * a, thread-safe, 0MQ context as an argument. The stop method will block
-     * until _all_ sockets within the context have been closed. Sockets blocking
-     * on a `recv` will be interrupted with ETERM upon context closure.
-     */
-    void start(faabric::transport::MessageContext& context);
-
-    void stop(faabric::transport::MessageContext& context);
```

Comment:
This is just removing the `start` and `stop` methods which took a context as argument.